packages/devtron-fe-lib/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@devtron-labs/devtron-fe-lib",
+  "version": "1.0.0",
+  "main": "index.js",
+  "types": "index.d.ts",
+  "scripts": {},
+  "author": "devtron-labs",
+  "license": "ISC"
```

Comment:
- remove license from here
- Sync with Nishant/Vivek and add the correct license at the root level

Reply:
Using apache-2.0. Added that to top most package.json and removed mention of license and author from all the descendant package.json(s)